python/run-tests.py

```diff
@@ -22,16 +22,16 @@
 from os import path
 
 
-def test(root_dir, test_path, packages):
-    # Run all of the tests under test_path directory, each of them
-    # has main entry point to execute, which is python's unittest testing
+def test(root_dir, codes_dir, packages):
```

Comment:
nit: 
1. I believe "code" is an uncountable noun, i.e. there is no plural.
2. Shall we just call it `toTest_dir`?

Reply:
@andreaschat-db Not sure if toTest is following the python naming convention.